enarx-keep-sgx-shim/src/start.S

```diff
@@ -162,7 +197,30 @@ syscall:
     mov     0x48(%rsp),             %rsp            # Get the exit context
 
     xor     %rcx,                   %rcx            # Clear %rcx
-    zerof   %rcx                                    # Clear CPU flags
+    zerof   %rcx                                    # Clear CPU flags 
```

Comment:
Why is this line showing up in the diff? I don't see any changes...

Reply:
It was an extra space issue; fixed it now using `git diff`.